build-systems/snap/snapcraft/snapcraft.yaml

```diff
@@ -15,6 +15,7 @@ apps:
     command: desktop-launch ${SNAP}/usr/bin/QOwnNotes -style=Fusion --snap
     environment:
         DISABLE_WAYLAND: 1
+        QT_QPA_PLATFORMTHEME: gtk3
```

Comment:
Thank you, but is there something that would enable some KDE look and feel instead of a GTK look and feel? 😅

Reply:
I don't know of any specific configuration for that, but as I understand it, this should enable the application to look more «at home» on any DE.
I don't use KDE, but if you do you can test some snaps with these changes already applied, like:
- https://snapcraft.io/keepassxc
- https://snapcraft.io/musescore

To see how they look like in there.

Reply:
I've keepassxc installed and all the dialogs are GTK dialogs, despite that it's a Qt app and I'm running KDE. I don't like that as a default-experience a lot. :grinning: 

They also use `QT_QPA_PLATFORMTHEME=gtk3` in <https://github.com/keepassxreboot/keepassxc/blob/develop/snap/local/launchers/gtk3-env-launch>

I think QOwnNotes currently is using the "Fusion" theme in snaps.
I almost would prefer different snaps for KDE and Gnome, like `qownnotes-kde` and `qownnotes-gtk`.
What's your opinion?

Reply:
> I almost would prefer different snaps for KDE and Gnome, like qownnotes-kde and qownnotes-gtk.
>  What's your opinion?

I haven't seen any snap doing that, so I don't really thought about it till now...I'm not really sure it's a good idea.

> Or is there another option to incorporate different look and feels?

Not that I know, but that is a question better asked on the forum: https://forum.snapcraft.io/ do you mind opening a tread?

Reply:
Phew, there are a lot of snaps out there using many integrations. That topic needs more research.

Reply:
Nevertheless I asked https://forum.snapcraft.io/t/kde-and-gtk-integration-for-qt-app/15892